internal/server/data/identity.go

```diff
@@ -157,7 +157,7 @@ func DeleteIdentities(tx GormTxn, selectors ...SelectorFunc) error {
 			return err
 		}
 
-		groups, err := ListGroups(tx, nil, ByGroupMember(i.ID))
+		groups, err := ListGroups(tx, ListGroupsOptions{ByGroupMember: i.ID})
```

Comment:
This was a required selector, but the value is always non-zero because it comes from `ListIdentities`.